draft-ietf-rtcweb-jsep.xml

```diff
@@ -2270,15 +2269,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             MUST be used, even if ICE has not yet completed. Note that
             this differs from the guidance in
             <xref target="RFC5245" />, Section 9.1.2.2, which only
-            refers to offers created when ICE has completed. Each
-            "a=rtcp" attribute line MUST also be filled in with the
-            port and address of the appropriate default candidate,
-            either the default RTP or RTCP candidate, depending on
-            whether RTCP multiplexing is currently active or not. Note
-            that if RTCP multiplexing is being offered, but not yet
-            active, the default RTCP candidate MUST be used, as
-            indicated in
-            <xref target="RFC5761"></xref>, section 5.1.3. In each
+            refers to offers created when ICE has completed. In each
```

Comment:
"of the desired type" is no longer accurate now that we aren't discussing a=rtcp.

Suggest adjusting the text to say

"In each case, if no **RTP candidates** have yet been gathered, dummy values MUST be used, as described above."